test/evaluator/stlc.t

```diff
@@ -39,48 +39,228 @@ Shadowing x
   λa. a
 
 Church numerals
+$ mtt eval <<EOF
+> let n0 = λf:F. λx:X . x in
+> let n1 = λf:F. λx:X . f x in
+> let n2 = λf:F. λx:X . f (f x) in
+> let n3 = λf:F. λx:X . f (f (f x)) in
+> let n4 = λf:F. λx:X . f (f (f (f x))) in
+> 
+> let true = λx:A. λy:B . x in
+> let false = λx:A . λy:B . y in
+> let if = λp:P. λt:A. λe:B. (p t) e in
+> 
+> let pair = λa:A . λb:B . λt:A -> B -> C . (t a) b in
+> let fstt = λp:A -> B -> C. p (λx:A. λy:B . x) in
+> let sndd = λp:A -> B -> C. p (λx:A . λy:B . y) in
+> 
+> let succ = λn:N. λf:F. λx:X. f ((n f) x) in
```

Comment:
One more thing: we should keep this evaluator test: `succ` is a keyword now and thus cannot be used as an identifier. We just need to rename `succ` to something else, like `suc` and the test should pass.

Reply:
done